.github/workflows/release.yml

```diff
@@ -45,6 +45,12 @@ jobs:
           MAVEN_CENTRAL_TOKEN: ${{ secrets.SONATYPE_PASSWORD }}
           MAVEN_GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
 
+      - name: Create GitHub release
+        uses: softprops/action-gh-release@v0
```

Comment:
Does `v0` automatically uses the latest version? Automatic release notes seems to have been added in the latest version [v0.1.14](https://github.com/softprops/action-gh-release/releases/tag/v0.1.14).

Reply:
Yes, just as how you reference the other actions in the workflow file, i.e. via release tags and major versions. This will automatically resolve the neweset v0.MINOR.PATCH release. (See also: https://docs.github.com/en/actions/creating-actions/about-custom-actions#good-practices-for-release-management)

Reply:
Hmm, looks like `v0` does not exist?
https://github.com/gluonhq/gluonfx-maven-plugin/runs/5813025436?check_suite_focus=true#step:1:31

Reply:
https://github.com/marketplace/actions/gh-release show `v1`

Reply:
Ok, that works: https://github.com/gluonhq/gluonfx-maven-plugin/releases/tag/1.0.13

Reply:
Ah, I overlooked [this](https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-tags-for-release-management):

>Move the major version tag (such as v1, v2) to point to the Git ref of the current release. For more information, see "[Git basics - tagging](https://git-scm.com/book/en/v2/Git-Basics-Tagging)."

That is, Action authors shall move the corresponding major version tag (here `v0`) to the current (matching major) release. Since only `v0.<minor>.<patch>` tags exist, this didn't work. (I always thought this was a matching mechanism…)

Anyway, looks like the release worked out @jperedadnr?

Reply:
yes, the release worked fine with v1.